File: dart_examples/flutter_auth_example/lib/altogic.dart
import 'package:altogic/altogic.dart';

const String envUrl = 'https://c4-na.altogic.com/e:63627456fe0ce760f2900095';
const String clientKey = '5b05db9ed41044049441b00885c262e8';
const String clientKeyy = String.fromEnvironment("name");

final altogic = createClient(envUrl, clientKey);

File: examples/authentication/Ionic Vue/src/libs/altogic.ts
import { createClient } from 'altogic';

const ENV_URL = 'https://pn45-90sr.c1-europe.altogic.com';
const CLIENT_KEY = '7427b15c87f34ad9a80e16a382d7fd06';

const altogic = createClient(ENV_URL, CLIENT_KEY, {
	signInRedirect: '/login',
});

export default altogic;
